src/Microsoft.OData.Core/UriParser/SemanticAst/AnnotationSegment.cs

```diff
@@ -0,0 +1,95 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="AnnotationSegment.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData.UriParser
+{
+    #region Namespaces
+
+    using Edm.Vocabularies;
+    using Microsoft.OData.Edm;
+
+    #endregion Namespaces
+
+    /// <summary>
+    /// A segment representing an Annotation
+    /// </summary>
+    public sealed class AnnotationSegment : ODataPathSegment

```

Comment:
>AnnotationSegment [](start = 24, length = 17)

Maybe it's better to name it as "TermSegment" ?

Reply:
Well, the term is what is defined in metadata, the annotation is the application of the term.  In this case, what is being projected is not the term definition, but the application of the term to the data (i.e., the value of the term, rather than the definition.

---
In reply to: [166081172](https://github.com/OData/odata.net/pull/1063#discussion_r166081172) [](ancestors = 166081172)